apps/battlepass-service/.dev.vars.example

```diff
@@ -0,0 +1,3 @@
+SUPABASE_SERVICE_KEY=PLACEHOLDER
```

Comment:
Change these environment variable names in .dev.vars.example to reflect what you're using in the code